src/PhpParser/NodeTraverser/RectorNodeTraverser.php

```diff
@@ -65,12 +70,18 @@ public function getVisitorsForNode(Node $node): array
         $nodeClass = $node::class;
         if (! isset($this->visitorsPerNodeClass[$nodeClass])) {
             $this->visitorsPerNodeClass[$nodeClass] = [];
+            if ($this->nameResolver instanceof NameResolver) {
```

Comment:
NameResolver is already executed on parsing content process, so should not needed here.

Reply:
If you can tell me where I can get the FQN for a `Node` or the current file, I'd be happy to remove the resolver again. But without it, `Node::namespaceName` is empty.